### ISSUE TITLE
Using Form.Input creates 2 nested fields

### ISSUE THREAD

**Issue by waldobronchart** [zabute/formsy-semantic-ui-react#5]
The following code
```
const errorLabel = <Label basic pointing />
...
<Form.Input type='text' name='username' placeholder='E-mail address' icon='at' 
  required
  errorLabel={errorLabel}
  validations='isEmail'
  validationErrors={{
    isEmail: 'This is not a valid e-mail address',
  }} />
```

Produces this in the DOM
```
<div class="field">
  <div class="error required field">
    <div class="ui icon input">
      <input type="text" value="sdfsdf" name="username" placeholder="E-mail address" required="">
      <i aria-hidden="true" class="at icon"></i>
    </div>
  </div>
  <div class="ui pointing basic label">This is not a valid e-mail address</div>
</div>
```

Notice the `<div class="error required field">` in the `<div class="field">`

This creates unwanted margin between the field and the validation error label

**Comment by zabute** [comment]
fixed in ```0.2.2```

**Comment by waldobronchart** [comment]
Thanks for the fix!
